Makefile

```diff
@@ -57,7 +57,7 @@ license:
 	GOBIN=$(GOBIN) scripts/update-license.sh
 
 lint:
-	(which golangci-lint || go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.22.2)
+	(which golangci-lint || go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.31.0)
```

Comment:
Why not get the latest always ?

Reply:
Upgrading the linter can lead to new lint errors. We should avoid random PRs getting build errors.

Reply:
Okay